application/views/admin/update/_updateContainer.php

```diff
@@ -16,40 +16,38 @@
 
 <!-- this view contain the input provinding to the js the inforamtion about wich content to load : check buttons or comfortupdate -->
 <?php
-    $this->renderPartial("./update/_ajaxVariables");
+$this->renderPartial("./update/_ajaxVariables");
 ?>
+<div class="container-fluid">
+    <div class="col-lg-12 list-surveys" id="comfortUpdateGeneralWrap">
+        <h3>
+            <span id="comfortUpdateIcon" class="icon-shield text-success"></span>
+            <?php if (YII_DEBUG): ?>
+                <small>Server:<em class="text-warning"> <?php echo Yii::app()->getConfig("comfort_update_server_url"); ?></em></small>
+            <?php endif; ?>
+        </h3>
 
-<div class="col-lg-12 list-surveys" id="comfortUpdateGeneralWrap">
-    <h3>
-        <span id="comfortUpdateIcon" class="icon-shield text-success"></span>
-        <?php if(YII_DEBUG):?>
-            <small>Server:<em class="text-warning"> <?php echo Yii::app()->getConfig("comfort_update_server_url");?></em></small>
-        <?php endif;?>
-    </h3>
+        <div class="row">
+            <div class="col-lg-12 content-right">
+                <div id="updaterWrap">
+                    <div id="preUpdaterContainer">
+                        <!-- The check buttons : render by ajax only if no step is required by url or post -->
+                        <?php // $this->renderPartial("./update/check_updates/_checkButtons", array( "thisupdatecheckperiod"=>$thisupdatecheckperiod, "updatelastcheck"=>$updatelastcheck,"UpdateNotificationForBranch"=>$UpdateNotificationForBranch )); ?>
```

Comment:
Is this an left over? If this would be an left over, please remove it. Otherwise write it into the comment, that you should enable this (if its needed). Otherwise it looks like disabled code.

Reply:
I can check what this is, but haven't touched that part of the code